index.html

```diff
@@ -250,7 +267,10 @@
         togglemathjax(latexenabled);
         e.preventDefault();
         return false;
-    }  
+    }
+    else if (e.keyCode == 79 && e.ctrlKey && e.shiftKey) {    // CTRL + SHIFT + O

```

Comment:
Create a constant for whatever key value 79 is. Makes it easier when reading if we know what key that is. No need for the comment after that.

Reply:
idem, we'll have very few commands at the end (minimalism), so for now let's keep it like that (only one line to read) instead of splitting this into several lines (need to scroll to see the constant, then scroll back to the `if`...)

Reply:
Constants should have a name that describe exactly what they are so you don't have to scroll up or down to know what they're doing.

Reply:
@lodenrogue these changes are not relevant to my pull request. Can you please either create a new issue instead or a pull request with your changes?

Reply:
These comments are in regards to the code you added. They are relevant to this pull request.

Reply:
@lodenrogue I understand your point of view, but I suggest that we keep these minor modifications for later if the project really gets longer code. I think a constant is really mandatory if the same value is (re)used at least twice. Here it's not so mandatory ;)

Reply:
It's your product. You are the final arbitrator but just keep in mind that some senior engineers will find issues with this coding standard because we've seen the negative effects it can have on a long term project.

Reply:
Thanks for this advice @lodenrogue, I'll think about it, and maybe one day I'll take this (good) habit of using more constants.

Reply:
@lodenrogue just FYI if you look at the rest of the file, all of the keyboard commands are done like this. I was just following the existing style while adding this feature.

Reply:
Yes. We should try to leave the sandbox a little cleaner than we found it every time we play.

Reply:
@lodenrogue it's just different coding practice, both are good :) Let's respect both ways of doing it :)

Reply:
No one is disrespecting anything. We're just having a healthy discussion about best practices. Are we not allowed to discuss this further on this repo?

Reply:
It's ok of course :) but not sure it's related to this (now merged) pull request. Thanks for your advice. I will take your point of view in consideration for next commits.